apps/src/applab/applab.js

```diff
@@ -103,6 +106,18 @@ function loadLevel() {
   Applab.softButtons_ = level.softButtons || {};
   Applab.appWidth = level.appWidth || defaultAppWidth;
   Applab.appHeight = level.appHeight || defaultAppHeight;
+  // In share mode we need to reserve some number of pixels for our in-app
+  // footer. We do that by making the play space slightly smaller elsewhere.
+  // Applab.appHeight represents the height of the entire app (footer + other)
+  // Applab.footerlessAppHeight represents the height of only the "other"
+  if (Applab.appHeight > 480) {
+    throw new Error('Strange things may happen with appHeight > 480');
+  }
+  if (Applab.appHeight === 480) {
+    Applab.footerlessAppHeight = Applab.appHeight - FOOTER_HEIGHT;
+  } else {
+    Applab.footerlessAppHeight = Applab.appHeight;
```

Comment:
It's not clear what we should do in terms of the footer for heights > 480. Rather than have unexpected behavior for, I decided to explicitly disallow it. If we decide we need > 480, we'll get an exception and then think about what should happen in terms of the footer in this scenario.